CHANGELOG.md

```
@@ -1,5 +1,8 @@
 [comment]: <> (All notable changes to this project will be documented in this file.)
 
+# 0.19.0
+### Breaking Changes:
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)